examples/syscall-steal.c

```diff
@@ -43,6 +43,14 @@
 
 #if defined(CONFIG_KPROBES)
 #define HAVE_KPROBES 1
+#if defined(CONFIG_X86_64)
+ /* For x86 architecture, the syscall table cannot be used to invoke a syscall
+ * after commit 1e3ad78.
+ * Use a hook on the syscall entry instead to intercept the syscall.
+ * Alternatively, set USE_KPROBES_PRE_HANDLER_BEFORE_SYSCALL to 0 to use the old method.
+ */
+#define USE_KPROBES_PRE_HANDLER_BEFORE_SYSCALL 1
```

Comment:
You should set `USE_KPROBES_PRE_HANDLER_BEFORE_SYSCALL` to zero for the opposite condition.